cryptofeed/exchange/huobi.py

```diff
@@ -74,7 +74,7 @@ async def _trade(self, msg: dict, timestamp: float):
 
     async def message_handler(self, msg: str, timestamp: float):
         # unzip message
-        msg = zlib.decompress(msg, 16 + zlib.MAX_WBITS)
+        msg = zlib.decompress(msg, 16 + zlib.MAX_WBITS).decode(encoding='utf-8', errors='strict')
```

Comment:
looks like the library has been updated to do byte parsing, so you should be able to remove this and require the latest version of the library in setup.py